### PR TITLE
TRUNK-5810:replacing Relationship class hibernate mappings with annot…

### PR DESCRIPTION
**Pull request by joelss-tech**
…ations

## Description of what I changed
1. Removed the Relationship.class xml files and mappings.
2. Added annotations.

## Issue I worked on
https://issues.openmrs.org/browse/TRUNK-5810

## Checklist: I completed these to help reviewers :)

- [ ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: api/src/main/java/org/openmrs/Relationship.java [hunk 1]
```diff
@@ -11,25 +11,48 @@
...
+   @Table(name = "relationship")
 public class Relationship extends BaseChangeableOpenmrsData {
 	
 	public static final long serialVersionUID = 323423L;
 	
 	// Fields
-	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name="relationship_id",nullable=false)
```
Comment (tendomart) [inline api/src/main/java/org/openmrs/Relationship.java:37]: This comma should be changed
`
@Column(name="relationship_id", nullable=false)`

File: api/src/main/java/org/openmrs/Relationship.java [hunk 2]
```diff
@@ -11,25 +11,48 @@
...
 	private Integer relationshipId;
-	
+
+	@ManyToOne
+	@JoinColumn(name = "person_a", nullable=false)
```
Comment (tendomart) [inline api/src/main/java/org/openmrs/Relationship.java:41]: `@JoinColumn(name = "person_a",  nullable=false)`

same here

File: api/src/main/java/org/openmrs/Relationship.java [hunk 3]
```diff
@@ -11,25 +11,48 @@
...
 	private Person personA;
-	
+
+	@ManyToOne
+	@JoinColumn(name="relationship", nullable=false)
 	private RelationshipType relationshipType;
-	
+
+	@ManyToOne
+	@JoinColumn(name="person_b",nullable = false)
```
Comment (tendomart) [inline api/src/main/java/org/openmrs/Relationship.java:49]: and here

File: api/src/main/java/org/openmrs/Relationship.java [hunk 4]
```diff
@@ -11,25 +11,48 @@
...
 	private Person personB;
-	
+
+	@Column(name= "start_date",length=19)
```
Comment (tendomart) [inline api/src/main/java/org/openmrs/Relationship.java:52]: and here

File: api/src/main/java/org/openmrs/Relationship.java [hunk 5]
```diff
@@ -11,25 +11,48 @@
...
 	private Date startDate;
-	
+
+	@Column(name="end_date",length= 19)
```
Comment (tendomart) [inline api/src/main/java/org/openmrs/Relationship.java:55]: @Column(name="end_date",length= 19)

and here

File: api/src/main/java/org/openmrs/Relationship.java (hunk 1 above)
Comment (joelss-tech) [reply api/src/main/java/org/openmrs/Relationship.java:37]: idin't get you Mr Tendo,atleast a hint about how i should change the comma.can i just get rid of it + the nullable........?

Comment (mozzy11) [reply api/src/main/java/org/openmrs/Relationship.java:37]: @joelss-tech , 
the long term solution is to setup the [OpenMRS formatter](https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+Eclipse#DeveloperHowToSetupAndUseEclipse-XMLCodeFormat) for your IDE.
you can set it up in Ecclipse, IntelliJ  or VScode

**Comment by joelss-tech** [comment]
@mozzy11 ,come and review this

**Pull request closed without merging**